code/controllers/TaxonomyDirectoryController.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * Class TaxonomyDirectoryController
+ *
+ * Controller for returning a list of pages tagged with a specific Taxonomy Term
+ */
+class TaxonomyDirectoryController extends Controller
+{
+
+    private static $allowed_actions = array(
+        'index'
+    );
+
+    public function index(SS_HTTPRequest $HTTPRequest)
+    {
+        $termString = $HTTPRequest->param('ID');
+
+        $pages = Page::get()
+            ->innerJoin(
+                'BasePage_Terms',
+                '"Page"."ID"="BasePage_Terms"."BasePageID"')
+            ->innerJoin(
+                'TaxonomyTerm',
+                "\"BasePage_Terms\".\"TaxonomyTermID\"=\"TaxonomyTerm\".\"ID\" AND \"TaxonomyTerm\".\"Name\" = '$termString'");
+
+        return $this->customise(new ArrayData(array(
```

Comment:
A Title here would be handy, perhaps configurable also in yaml?

Reply:
Done, but not configurable